src/libraries/System.Runtime/tests/LinkerTests/System.Runtime.LinkerTests.proj

```diff
@@ -0,0 +1,6 @@
+<Project DefaultTargets="Build">
+  <Import Project="$([MSBuild]::GetPathOfFileAbove(Directory.Build.props))" />
+
+  <Import Project="$(RepositoryEngineeringDir)linkerTests/linkerCorrectnessTests.targets" />
```

Comment:
Should we put this in a common place instead? We could detect that it is a linker test if the project ends with `.LinkerTests`

Reply:
Sure, I don't have a strong opinion here so I'll move it to a central place instead.